x-pack/solutions/security/plugins/security_solution/public/detections/components/attacks/table/default_group_title_renderers/attack_group_content.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiTitle } from '@elastic/eui';
+import {
+  replaceAnonymizedValuesWithOriginalValues,
+  type AttackDiscoveryAlert,
+} from '@kbn/elastic-assistant-common';
+
+import { AttackDiscoveryMarkdownFormatter } from '../../../../../attack_discovery/pages/results/attack_discovery_markdown_formatter';
+
+export const ATTACK_TITLE_TEST_ID = 'attack-title' as const;
+export const ATTACK_DESCRIPTION_TEST_ID = 'attack-description' as const;
```

Comment:
You're already passing `attack` with the `dataTestSubj` prop, so these could be just
```typescript
export const ATTACK_TITLE_TEST_ID = '-title' as const;
export const ATTACK_DESCRIPTION_TEST_ID = '-description' as const;
```